### PR TITLE
Filter by label in all possible places

### PR DESCRIPTION
**Pull request by sanbornsen**
Fix: https://github.com/openshiftio/openshift.io/issues/329

### CODE REVIEW

**Comment by fabric8cd** [comment]
@sanbornsen fabric8/fabric8-ui:SNAPSHOT-PR-2208-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2208-fabric8-planner.badger.fabric8.io

File: src/app/components/toolbar-panel/toolbar-panel.component.ts [hunk 1]
```diff
@@ -274,12 +282,28 @@ export class ToolbarPanelComponent implements OnInit, AfterViewInit, OnDestroy {
     let recentAppliedFilters = {};
     $event.appliedFilters.forEach((filter) => {
       if (filter.query.id !== 'loader') {
-        recentAppliedFilters[filter.field.id] = filter;
+        if (Object.keys(recentAppliedFilters).indexOf(filter.field.id) === - 1) {
```
Comment (vikram-raj) [inline src/app/components/toolbar-panel/toolbar-panel.component.ts]: extra space between - and 1.

**Comment by fabric8cd** [comment]
@sanbornsen fabric8/fabric8-ui:SNAPSHOT-PR-2208-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2208-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@sanbornsen fabric8/fabric8-ui:SNAPSHOT-PR-2208-3 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2208-fabric8-planner.badger.fabric8.io

File: src/app/components/planner-list/planner-list.component.ts [hunk 2]
```diff
@@ -726,4 +727,16 @@ export class PlannerListComponent implements OnInit, AfterViewInit, DoCheck, OnD
   togglePanel() {
     this.sidePanelRef.toggleSidePanel();
   }
+
+  onClickLabel(event) {
```
Comment (vikram-raj) [inline src/app/components/planner-list/planner-list.component.ts]: closing bracket '}' for this function is missing.

**Comment by fabric8cd** [comment]
@sanbornsen fabric8/fabric8-ui:SNAPSHOT-PR-2208-5 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2208-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@sanbornsen fabric8/fabric8-ui:SNAPSHOT-PR-2208-6 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2208-fabric8-planner.badger.fabric8.io

**Pull request merged**